### PR TITLE
Search bar

### PR DESCRIPTION
**Pull request by Nikolas-T**

### CODE REVIEW

**Comment by Nikolas-T** [comment]
some of Tyler's changes that appear to have been removed simply appear somewhere else in the code, the website does reflect all the recent changes made

File: index.html [hunk 1]
```diff
@@ -33,15 +33,17 @@ <h2 class="subtitle" id="hero-subtitle">
     </header>
 
     <main>
-        <section class="section">
```
Comment (RflctnOfU) [inline index.html:36]: why did you remove the section element?  Semantically it is better to have it.

File: assets/css/style.css [hunk 2]
```diff
@@ -84,7 +93,8 @@ body {
     max-width: 12%;
 }
 
-.my-list-container {
+/* turn this back into a container  */
+.my-list-contaxiner {
```
Comment (RflctnOfU) [inline assets/css/style.css:97]: this typo needs to be fixed

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (RflctnOfU) [inline .vscode/settings.json:1]: I have no idea what this file is...I don't think it should be here.

File: assets/css/style.css [hunk 4]
```diff
@@ -1,4 +1,6 @@
 @import "https://cdn.jsdelivr.net/npm/bulma@0.9.4/css/bulma.min.css";
+@import url('https://fonts.googleapis.com/css?family=Inconsolata:700');
+@import url("https://fonts.googleapis.com/css?family=Inconsolata:700");
```
Comment (RflctnOfU) [inline assets/css/style.css:4]: I have reviewed the merge request with Tyler and we are suggesting the following changes.  (overall, the code looks okay, but it is disorderly an example, the footer stylings should be at the bottom).

More individual comments lower.

File: assets/css/style.css [hunk 5]
```diff
@@ -70,6 +73,12 @@ body {
 
 #search-icon {
     width: 25px;
+    position: absolute;
+    width: 50%;
+    height: 50%;
+    margin-left: 15px;
+    margin-top: 15px;
+
 }
```
Comment (RflctnOfU) [inline assets/css/style.css:83]: perhaps the code you wrote can go here?

File: assets/css/style.css [hunk 6]
```diff
@@ -194,11 +204,180 @@ body {
...
+    margin-top: 15px;
+}
+
+.columns {
+    margin-left: 10%;
+    margin-top: 2%;
+}
+
+/* search bar design functionality */
+
```
Comment (TDEvans97) [inline assets/css/style.css:228]: You could create classes specific to the search bar rather than staying broad with .container, .box, *, etc.. This will help prevent overwriting CSS styling elsewhere on the page. Even if it looks okay with these changes currently, it is not very efficient and holds us back when adding more features and pages in the future.
